Gruntfile.js

```diff
@@ -43,16 +43,31 @@ module.exports = function (grunt) {
 		sauceLoginFile: grunt.file.exists('SAUCE_API_KEY.yml') ? grunt.file.readYAML('SAUCE_API_KEY.yml') : undefined,
 		sauceUser: process.env.SAUCE_USERNAME || 'fuelux',
 		sauceKey: process.env.SAUCE_ACCESS_KEY ? process.env.SAUCE_ACCESS_KEY : '<%= sauceLoginFile.key %>',
-		allTestUrls: ['2.1.0', '1.11.0', '1.9.1', 'browserGlobals'].map(function (ver) {
-			if (ver === 'browserGlobals') {
-				return 'http://localhost:<%= connect.testServer.options.port %>/test/fuelux-browser-globals.html';
+		// TEST URLS
+		distTestUrl: ['http://localhost:<%= connect.testServer.options.port %>/test/?testdist=true'],
+		allTestUrls: ['2.1.0', '1.11.0', '1.9.1', 'browserGlobals', 'codeCoverage' ].map(function (type) {
+			if (type === 'browserGlobals') {
+				return 'http://localhost:<%= connect.testServer.options.port %>/test/browser-globals.html';
+			}
+			else if (type === 'codeCoverage') {
+				return 'http://localhost:<%= connect.testServer.options.port %>/test/?coverage=true';
+			}
+			else {
+				// test dist with multiple jQuery versions 
+				return '<% distTestUrl %>'; //'http://localhost:<%= connect.testServer.options.port %>/test/?jquery=' + ver + '&testdist=true';
 			}
-
-			return 'http://localhost:<%= connect.testServer.options.port %>/test/fuelux.html?jquery=' + ver + '&testdist=true';
 		}),
-		testUrl: ['http://localhost:<%= connect.testServer.options.port %>/test/fuelux.html?jquery=' + '1.9.1&testdist=true'],
 
 		//Tasks configuration
+		blanket_qunit: {
+			source: {
+				options: {
+					urls: ['http://localhost:<%= connect.testServer.options.port %>/test/?coverage=true&gruntReport'],
+					threshold: 68,
+					globalThreshold: 81
```

Comment:
I always use `--force` when developing anyway, because if I save the file mid-line, it's guaranteed to fail. I will remove `?coverage=true` from `grunt servefast` if you want to run coverage and then you should run `grunt test` when you are done. How's that?